### PR TITLE
Fix and Doc updates

### PR DESCRIPTION
**Pull request by daniel-butler-irl**
### Description

fix: https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/issues/374
Cleanup: https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/issues/378

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: scripts/get_config_map_status.sh [hunk 1]
```diff
@@ -15,6 +15,8 @@ done
 
 if [[ $COUNTER -eq $MAX_ATTEMPTS ]]; then
   echo "ConfigMap '$CONFIGMAP_NAME' did not become available within $MAX_ATTEMPTS attempts."
+  #  Output for debugging
```
Comment (vburckhardt) [inline scripts/get_config_map_status.sh:18]: probably want to remove this before merging?

Comment (daniel-butler-irl) [reply scripts/get_config_map_status.sh:18]: I think leave it, if this code gets hit the module will fail and it could provide useful debug info

**Review by vburckhardt — APPROVED** [review]

**Pull request merged**